lib/ui/window/platform_configuration.cc

```diff
@@ -196,80 +196,109 @@ PlatformConfiguration::PlatformConfiguration(
 PlatformConfiguration::~PlatformConfiguration() {}
 
 void PlatformConfiguration::DidCreateIsolate() {
-  library_.Set(tonic::DartState::Current(),
-               Dart_LookupLibrary(tonic::ToDart("dart:ui")));
+  auto* library = Dart_LookupLibrary(tonic::ToDart("dart:ui"));
```

Comment:
The engine typically declares Dart handles explicitly as type `Dart_Handle`

Reply:
Updated